### PR TITLE
refactor: Upgrade website-3d-cell-viewer

### LINKED ISSUE #163: update cfe to use latest w3cv

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Problem
=======
Closes #163, upgrading CFE to use the latest version of website-3d-cell-viewer.

*Estimated review size: small, 10 minutes*

NOTE: This build will be broken until #166 is merged into this branch, due to dependency issues with Ant.

Solution
========
- Updates web-3d-viewer to version 2.9.0.
- Updates the API for the web-3d-viewer.
- Fixes some broken styling caused by Ant version mismatch. (Will be removed in #166.)

## Type of change
* Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by toloudis** [comment]
I think this needs the other PR to go through first so the build isn't broken?

**Review by toloudis — APPROVED** [review]
I'm pre-approving but this prob needs testing after the bigger changes in cfe are merged

**Review by interim17 — APPROVED** [review]

**Pull request merged**
